can_tools/scrapers/official/MD/md_state.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+
+import us
+from can_tools.scrapers import CMU, DatasetBase
+from can_tools.scrapers.official.base import ArcGIS
+
+
+class Maryland(ArcGIS, DatasetBase):
+    """
+    Fetch county level covid data from Maryland's ARCGIS dashboard
+    """
+
+    ARCGIS_ID = "njFNhDsUCentVYJW"
+    has_location = True
+    location_type = "county"
+    state_fips = int(us.states.lookup("Maryland").fips)
+    source = "https://coronavirus.maryland.gov/"
+
+    def fetch(self):
+        return self.get_all_jsons(
+            "MDH_COVID_19_Dashboard_Feature_Layer_Counties_MEMA", 0, ""
+        )
+
+    def normalize(self, data):
+        df = self.arcgis_jsons_to_df(data)
+        df.columns = [x.lower() for x in list(df)]
+        df["location"] = (self.state_fips * 1000) + df["county_fip"].astype(int)
+
+        crename = {
+            "totalcasecount": CMU(
+                category="cases", measurement="cumulative", unit="people"
+            ),
+            "totaldeathcount": CMU(
+                category="deaths", measurement="cumulative", unit="people"
+            ),
+            "total_pop_tested": CMU(
+                category="unspecified_tests_total",
+                measurement="cumulative",
+                unit="unique_people",
+            ),
+            "total_testing_vol": CMU(
```

Comment:
Is there any documentation we can find on what type of tests have been administered?

@cc7768 do you have any insight into this

Reply:
Hi Spencer!

When I first did some research into MD ArcGIS statistics, I made a little log of what we have available.  I didn't see testing types broken-out in any supporting tables (below), and I didn't see a way to do the breakout on the Maryland master dashboard (https://coronavirus.maryland.gov/).  I can try to pull in other sources - HRSA has some testing breakout numbers (https://bphc.hrsa.gov/emergency-response/coronavirus-health-center-data/md).  But maybe this should be another scraper entirely?  I didn't want to pollute this one with auxiliary sources.


MD ArcGIS FINDINGS:

Maryland ARCGIS Databases:  (https://services.arcgis.com/njFNhDsUCentVYJW/arcgis/rest/services/)
1.   MDCOVID19_DailyTestingVolumeByCounty:  Only stores the daily (not cumsum) volume of tests for every day in the last 6mo (every day a column)
2.   MDCOVID19_CasesByCounty:  No suitable information about testing - daily case cumsum by county (every row a day, every column a county)
3.   MDCOVID19_NumberofCasesByAffected:   Number of positive cases by governmental physical location within county --> not a complete county set of locations (primarily correctional facilities)
4.   MDCOVID19_TestingVolume:    State-wide (not broken-out by county) cumsum of tests (no test subtype) and number of positives by date (every row a day)
5.   MDCOVID19_TotalCasesStatewide:  Same as above, but only couints the cumsum of positives state-wide
6.   MDCOVID19_TotalPopulationTestedByCounty:   Same as #1, but now a cumsum statistic
7.   MDCOVID19_VolumnTesting:   Same as #4, but no longer a cumsum, now per-day totals
8.   MD_COVID19_Testing_Sites:   List of testing sites, no test counts.

Reply:
They have some [documentation](https://phpa.health.maryland.gov/Documents/faq_covid19_data_dashboard_083120pdf.pdf) on data definitions -- It looks like all the testing that's reported comes from PCR tests.

I haven't been able to figure out whether they are counting specimens or test encounters though

Reply:
FWIW, it seems that [CTP](https://covidtracking.com/data/state/maryland) thinks they are reporting in units of specimens